### ISSUE TITLE
critical_frontend_hard_error_search_responses alert silenced

### ISSUE THREAD

**Issue by keegancsmith** [sourcegraph/sourcegraph#31119]
This seems to be a real regression on sourcegraph.com since wednesday. However, I've already spent a fair amount of time on it this weekend, so I am silencing so we can investigate on monday.

**Comment by keegancsmith** [comment]
#### Notes

We consistently have had high error rates. We don't seem to be creating alerts in some cases, instead we are returning errors. This started at 2022-02-07 19:16 UTC.

- https://ui.honeycomb.io/sourcegraph/datasets/search/result/7aw64noM6mG
- https://sourcegraph.com/-/debug/grafana/d/frontend/frontend?orgId=1&viewPanel=100011&time=1644632187000&time&from=now-7d&to=now

deploy-sourcegraph-cloud commits around the time it started:

    66c75c36d8b277493b180c6a83a9b22735a3a171 2022-02-07 18:43:25 +0000
    129652_2022-02-07_0d560445d13a
    
    e7a2e01044c3e7728669dc4706ad655ddcb8f4c4 2022-02-07 17:05:07 +0000
    129614_2022-02-07_8aa32e70b29a

`0d560445d13a` is also reported by grafana a few minutes before we see the alert.

    $ git log --format=oneline 8aa32e70b29a..0d560445d13a
    0d560445d13a887db7e58e0f5e8450fb90dc40f2 dev/release: do not create GitHub release if release is old (#30712)
    fe09a7270b0b47bf3ae1a6fd18ecc0294f16b9cb dev/sg: render 'sg ci preview' with nice markdown (#30724)
    280f31b41d4510c9159bfa6e84020b338f983fe9 dev/ci: add RunType for nightly release branch healthchecks (#30581)
    9e09363f99101c7bfdfdc1e338f12ef07edede42 remove lower-level limits when and/or jobs are enabled (#30735)
    e23f54570cbd6c8a7d8e4f51c564d953e3e07c91 dev/ci: render per-RunType and per-diff pipeline docs (#30532)
    afae15f9ba0e9263408f23da633ba0e1f5fa8dde Restructure revisions section stories and enable snapshotting (#30627)
    90974c69ad5dfd346e63d1a762a029520241557b Code Insights: Add dynamic code insight example for the in-product landing page (#30742)
    9c08a9a053e9f73b07b91989b3b30375ba09ffb4 LSIF Typed: minor additions to prepare typed->graph conversion tool (#30756)
    33e4844c47f182783a5960ff42bdbe5630ec2127 web: remove duplicate hook (#30757)

The only relevant looking commit is the remove lower-level limits. The rollout before this however had stuff to do with switching to our internal error pkg which feels more related.

    repo: hello type:commit ciao after:"2021-05-22T10:01:26Z"

I can easily reproduce on dot-com, but not in my local dev.

<span class="timestamp-wrapper"><span class="timestamp">[2022-02-12 Sat 06:03] </span></span> I've now spent two hours trying to fix this. Just gonna turn the alert off and look again on monday.

**Comment by keegancsmith** [comment]
![image](https://user-images.githubusercontent.com/187831/153816370-d3cd699d-650e-432f-a2c8-64ab54610b19.png)

Amazing. It seems this is no longer happening. We did a deploy this morning and its stopped! I dug in and it either #31082 or #31112 resolved this and where landed friday/saturday. When I tried to reproduce the issue locally I had those changes (not sure if both), which is why I couldn't reproduce! I think we didn't deploy all weekend due to the build being broken. One must always remember to try reproduce on the same commit, continuous deployment isn't always happening due to things like build breaks.

I've unsilenced the alert.
